src/edu/csus/ecs/pc2/ui/eventfeed/ServicesView.java

```diff
@@ -141,7 +141,7 @@ public void setContestAndController(final IInternalContest inContest, final IInt
             public void run() {
```

Comment:
Bug - silently fails if Options Show PC^2 Log button selected.
This NPE condition should be detected and a message should be shown to the user.
Exception in thread "AWT-EventQueue-0" java.lang.NullPointerException
        at edu.csus.ecs.pc2.core.InternalController.showLogWindow(InternalController.java:4303)
        at edu.csus.ecs.pc2.ui.OptionsPane.showLog(OptionsPane.java:151)
        at edu.csus.ecs.pc2.ui.OptionsPane$3.actionPerformed(OptionsPane.java:221)
        at javax.swing.AbstractButton.fireActionPerformed(AbstractButton.java:2022)
        at javax.swing.AbstractButton$Handler.actionPerformed(AbstractButton.java:2348)
        at javax.swing.DefaultButtonModel.fireActionPerformed(DefaultButtonModel.java:402)
        at javax.swing.DefaultButtonModel.setPressed(DefaultButtonModel.java:259)
        at javax.swing.plaf.basic.BasicButtonListener.mouseReleased(BasicButtonListener.java:252)
        at java.awt.Component.processMouseEvent(Component.java:6533)
        at javax.swing.JComponent.processMouseEvent(JComponent.java:3324)
        at java.awt.Component.processEvent(Component.java:6298)

Reply:
This dialog about log viewer disabled also is a visual cue that the PR has been merged.

Reply:
I have added the requested protection against NPE (along with a dialog informing the user what happened).

Reply:
This has been resolved/fixed.